charts/juicefs-s3-gateway/templates/deployment.yaml

```diff
@@ -29,10 +29,19 @@ spec:
           {{- if .Values.image.pullPolicy }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           {{- end }}
+          {{- if .Values.secret.token }}
+          securityContext:
+            privileged: true
```

Comment:
Gateway needs privileged?

Reply:
gateway itself needs no privilege, but `mountPropagation: Bidirectional` requires privilege

is there another solution to avoid privilege requirements?

Reply:
juicefs root dir do not need `bidirectional` here. `bidirectional` is meant to pass the mount point between the container and the host.

Reply:
I can put juicefs auth in the mount container, rather than the format container. this solves the issue?